src/app/pages/my-virtrolio/my-virtrolio.component.ts

```diff
@@ -74,6 +93,42 @@ export class MyVirtrolioComponent implements OnInit {
     }
   }
 
+  shareLink() {
+    if (this.canShare()) {
+      this.navigator.share({
+        title: this.displayName + '\'s virtrolio!',
+        text: 'Visit this link to sign their virtrolio and send them a custom message!',
+        url: this.link,
+      })
+        .then(() => console.log('Successful share'))
+        .catch((error) => console.log('Error sharing', error));
+    } else {
+      alert('Share not supported on this device. Use a mobile device!');
+    }
+  }
+
+  /**
+   * Share the user's sharing link on the appropriate platform
+   * @param platform - platform on which to post the sharing link
+   */
+  postOnSocial(platform: string) {
+    const urlFriendlyLink = 'https%3A//virtrolio.web.app/signing?uid=' + this.authService.uid() + '%26key=' +
+      this.link.match(/key=([^&]*)/)[1];
+
+    const bodyText = 'Send%20me%20a%20message!%20';
+
+    if (platform === 'facebook') {
+      window.open('https://www.facebook.com/sharer/sharer.php?u=' + urlFriendlyLink, '_blank');
+    } else if (platform === 'twitter') {
+      window.open('https://twitter.com/intent/tweet?url=' + urlFriendlyLink + '&text=' + bodyText, '_blank');
+    } else if (platform === 'email') {
+      window.open('mailto:?subject=' + 'Virtrolio%20-%20Online%20Yearbook%20Signing!'
+        + '&body=' + bodyText + urlFriendlyLink, '_blank');
```

Comment:
`%0D%0A` is a new line
```suggestion
        + '&body=' + bodyText + '%0D%0A' + urlFriendlyLink, '_blank');
```